bazel/repositories.bzl

```diff
@@ -211,9 +211,9 @@ def proxy_wasm_cpp_host_repositories():
         http_archive,
         name = "com_github_bytecodealliance_wasmtime",
         build_file = "@proxy_wasm_cpp_host//bazel/external:wasmtime.BUILD",
-        sha256 = "6ef70886da14245f575c6ff8c7c999ae22579257eba5ebf382e066598c1e381c",
-        strip_prefix = "wasmtime-0.39.1",
-        url = "https://github.com/bytecodealliance/wasmtime/archive/v0.39.1.tar.gz",
+        sha256 = "af9906ce0c30f3de2bc967342735809817e74875cf8b3f509efd297887f54a1c",
+        strip_prefix = "wasmtime-1.0.0",
+        url = "https://github.com/bytecodealliance/wasmtime/archive/v1.0.0.tar.gz",
```

Comment:
Would you mind bumping this to v1.0.1 (here and in `Cargo.toml`)?

Reply:
That's done now. Apologies it's taken so long. I thought this would be quick, then got caught up in a house move.

Reply:
That's done now. Apologies it's taken so long. I thought this would be quick, then got caught up in a house move.

Reply:
> That's done now.

It doesn't look like it is. This should be pointing to `v1.0.1.tar.gz`.

> Apologies it's taken so long. I thought this would be quick, then got caught up in a house move.

No worries, thanks for all the help!

Reply:
This is still `v1.0.0`.